### DISCUSSION TITLE
Block specific activities in non-root mode

### DISCUSSION THREAD

**Discussion by lihenggui** [lihenggui/blocker#467] [Ideas]
1. Show current activity in a floating window and disable them in just one clock
2. Send back pressed events the next time the user opens this activity.

**Comment by The-Nickname** [comment]
To me https://github.com/lihenggui/blocker?tab=readme-ov-file#shizukusui-mode is not quite clear, if Shizuku is used in non-root mode as far as I can tell it is not possible to block anything at all, correct?
So is there even a point in having Shizuku installed without root for blocker? Unless I modify the APKs myself I guess, or is there any feature that can be used with Blocker+Shizuku-non-root?
